src/config.js

```diff
@@ -78,6 +79,9 @@ const setConfig = async (cfg) => {
     config.ports.simulatorApi = cfg.SIMULATOR_API_LISTEN_PORT || config.ports.simulatorApi;
     config.ports.reportApi = cfg.REPORT_API_LISTEN_PORT || config.ports.reportApi;
     config.ports.testApi = cfg.TEST_API_LISTEN_PORT || config.ports.testApi;
+    if (cfg.PARTIES) {
```

Comment:
Just wondering why this looks different from the rest of the config. Maybe could be another conditional assignment?

Reply:
Good catch. It's because I didn't read around enough. Thanks.